### ISSUE TITLE
Testing does this work?

### ISSUE THREAD

**Issue by alerque** [simoncozens/GlyphSILE.glyphsplugin#1]
Am I reading this correctly that it's a copy of Lua's source code mashed on top of a copy of SILE's source code with some Objective C here and there gluing everything together?

If so, does it still work? And do you expect it to be useful in the future? And if so is there any way we could set it up with external module for SILE and automated testing to show if something we were doing upstream was going to affect it?

**Comment by simoncozens** [comment]
Yes, it works, and yes, you have it right. SILE and lua are brought in as submodules, but currently pinned to known-good versions. It's useful now and I hope it continues to be so. :-) It would be great to have it track SILE master, but if it ain't broke...

I am not sure how we can automate the testing of it, since it has to link against Glyphs' SDK, and then be opened from within Glyphs.
